provisioning_templates/snippet/kickstart_ifcfg_generic_interface.erb

```diff
@@ -37,10 +37,9 @@ PEERDNS=<%= primary %>
 PEERROUTES=<%= primary %>
 DEFROUTE=<%= primary %>
 <%- if @interface.primary -%>
-<%-   if !@dhcp && @subnet && @subnet.dns_primary.present? -%>
-<%=     "DNS1=\"#{@subnet.dns_primary}\"" %>
-<%-     if @subnet.dns_secondary.present? -%>
-<%=       "DNS2=\"#{@subnet.dns_secondary}\"" %>
+<%-   if !@dhcp && @subnet -%>
+<%-     @subnet.dns_servers.each_with_index { |nameserver, index| -%>
```

Comment:
I don't know about the compatibility of `each_with_index`.

Reply:
It's not allowed in safemode, please avoid it.